libs/pageserver_api/src/models.rs

```diff
@@ -48,13 +48,33 @@ pub enum TenantState {
 }
 
 impl TenantState {
-    pub fn has_in_progress_downloads(&self) -> bool {
+    pub fn attachment_status(&self) -> TenantAttachmentStatus {
+        use TenantAttachmentStatus::*;
         match self {
-            Self::Loading => true,
-            Self::Attaching => true,
-            Self::Active => false,
-            Self::Stopping => false,
-            Self::Broken { .. } => false,
+            // The attach procedure writes the marker file before adding the Attaching tenant to the layer map.
```

Comment:
> ... adding the Attaching tenant to the layer map

That doesn't make sense to me. You add layers to the layer map, not tenants

Reply:
Please resolve if 'tenants map' clarifies it for you.